packages/backend-app-api/config.d.ts

```diff
@@ -88,6 +88,47 @@ export interface Config {
                */
               subject: string;
             };
+            /**
+             * The scope of access permitted for this access method. If no scope
+             * is given, it'll have unlimited access. This access restriction
+             * applies for the framework level; individual plugins may have
+             * their own access control mechanisms on top of this.
+             */
+            scope?: {
+              /**
+               * If given, this method is limited to only having access to these
+               * plugins. Can be combined with `permissionAttributes`
+               * (especially in the form of `{ action: read }`), but it's
+               * superfluous to combine it with `permissions` that map to the
+               * same plugins.
+               */
+              plugins?: string[];
+              /**
+               * If given, this method is limited to only performing actions
+               * with these named permissions. It's superfluous to combine this
+               * with `plugins` that map to the same permissions, or with
+               * `permissionAttributes`.
+               *
+               * Note that this only applies where permissions checks are
+               * enabled in the first place. Endpoints that are not protected by
+               * the permissions system at all, are not affected by this
+               * setting.
+               */
+              permissions?: string[];
```

Comment:
Just to have the discussion: do you think it would make sense to configure permissions per plugin instead? One particularly good reason for that I think is that if you're not also restricting to certain plugins the token scope will still be very wide. If you just restrict to `catalog.entity.read` or w/e it is, then you'd still be allowed to call any other endpoint on any plugin that doesn't have permission integrations.